packages/bpk-component-scrollable-calendar/src/BpkScrollableCalendarGridList.tsx

```diff
@@ -75,6 +63,20 @@ const BpkScrollableCalendarGridList = (props: Props) => {
   const endDate = startOfDay(startOfMonth(rest.maxDate));
   const monthsCount = DateUtils.differenceInCalendarMonths(endDate, startDate);
 
+  // These constants are here to facilitate calculating the height
+  // Row and month item heights are defined in rem to support text scaling
+  const ROW_HEIGHT = 2.75;
+  // This is the additional height of each grid without any rows.
+  const BASE_MONTH_ITEM_HEIGHT = monthHeight;
```

Comment:
That makes sense.

I've updated it so that we're passing a prop for `ROW_HEIGHT`.

See [fa77fb8](https://github.com/Skyscanner/backpack/pull/3057/commits/fa77fb819bd476d530fd4a2b3427299d5a53d747)